programs/pkey/dh_client.c

```diff
@@ -20,6 +20,7 @@
 #include "mbedtls/build_info.h"
 
 #include "mbedtls/platform.h"
+#include "mbedtls/md.h"
```

Comment:
Ah, sorry, the CI points out I was wrong when I changed the macro to `MBEDTLS_MD_CAN_SHA1`, as the program is calling `mbedtls_sha1()` directly, so we should have kept `MBEDTLS_SHA1_C` in the check a few lines below.

(I believe `MBEDTLS_MD_CAN_SHA256` is correct though, as it's what's used with RSA, which uses MD light now. So, we need this include indeed - I'm only commenting on this line because that's the only new line for me to comment on.)

Note: same with `dh_server.c` I think.